Composer/packages/lib/ui-shared/src/components/ProvisionHandoff/ProvisionHandoff.tsx

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import { DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
+import { PrimaryButton } from 'office-ui-fabric-react/lib/Button';
+import formatMessage from 'format-message';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { IconButton } from 'office-ui-fabric-react/lib/components/Button';
+import { FontSizes, NeutralColors } from '@uifabric/fluent-theme/lib/fluent';
+import { useRef } from 'react';
+import { ITextField, TextField } from 'office-ui-fabric-react/lib/components/TextField';
+
+import { DialogTypes, DialogWrapper } from '../DialogWrapper';
+
+type ProvisionHandoffProps = {
+  title: string;
+  developerInstructions: string;
+  handoffInstructions: string;
+  hidden: boolean;
+  onBack: () => void;
+  onDismiss: () => void;
+};
+
+export const ProvisionHandoff = (props: ProvisionHandoffProps) => {
+  const textFieldRef = useRef<ITextField>(null);
+
+  const copyCodeToClipboard = () => {
+    try {
+      textFieldRef.current?.select;
```

Comment:
what does referencing the select property do here?

Reply:
I don't know! Patrick wrote this part.  It doesn't appear to be doing anything. I'll remove and ask him.